client/src/app/components/Pagination.tsx

```diff
@@ -40,6 +43,38 @@ class Pagination extends Component<PaginationProps, PaginationState> {
         }
     }
 
+    /**
+     * The component mounted.
+     */
+    public componentDidMount(): void {
+        super.componentDidMount();
```

Comment:
Explicit calls to super methods are needed to init _isMounted variable of the AsyncComponent class and it will not work correctly if removed. _isMounted is used to prevent leaking while setting State in resize() method.